### ISSUE TITLE
Adding more books in the app

### ISSUE THREAD

**Issue by kamalbuilds** [legendarykamal/Lets-go-shop#6]
Currently there are a minimum of just 5, we need more atleast a 10-12 to showcase.

**Comment by kamalbuilds** [comment]
@Omafovbe would you like to?

**Comment by Omafovbe** [comment]
So in fulfilling this task you need both image cover, price both on the home component as well as the basketReducer right?

**Comment by kamalbuilds** [comment]
yes , home ,cart and basket reducer . I have done the scaffold already you can just extend it for the rest of the books.
Its easy, I will help you if you get stuck anywhere :small_airplane:

**Comment by Omafovbe** [comment]
Thanks will let you know if I run into any issues.

**Comment by kamalbuilds** [comment]
should i get you assigned?

**Comment by Omafovbe** [comment]
Yes please do

**Comment by Omafovbe** [comment]
Got a report PR #37 was merge to the branch again and not into master. was this intended?

**Comment by kamalbuilds** [comment]
No, Its not merged to anything @Omafovbe

**Comment by Omafovbe** [comment]
Yes I would love to. Thanks

**Comment by Omafovbe** [comment]
Yes please do
